Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM php:8.1-apache
 #Install git
 EXPOSE 80
 # Modules that this needs
-RUN apt-get update && apt-get install -y git netcat && apt-get clean -y
+# RUN apt-get update && apt-get install -y git netcat && apt-get clean -y
+RUN apt-get update && apt-get install -y git netcat-openbsd && apt-get clean -y
```

Comment:
This is fine, I think this was split between `netcat-traditional` and `netcat-openbsd` and either should be fine here.